### ISSUE TITLE
Standardize matplotlib styles

### ISSUE THREAD

**Issue by justinGilmer** [mosdef-hub/reproducibility_study#74]
We should include a `matplotlibrc` file so we do not need to worry as much about the plot formatting. If anyone has a nice rc file already, id be happy to include something like that.

**Comment by daico007** [comment]
[This](https://gist.github.com/daico007/a8c3ff2a7e7553ba961d9100eb65b01c) is the `matplotlibrc` that I am currently using for my paper.

**Comment by jennyfothergill** [comment]
@daico007 that looks good, but is [something missing at the end?](https://gist.github.com/daico007/a8c3ff2a7e7553ba961d9100eb65b01c#file-matplotlibrc-L229-L231) Forgive me if I am missing something obvious--I'm not super familiar with gists.

**Comment by daico007** [comment]
@jennyfothergill good catch! I made a mistake when doing the copy-paste, edited the gist to have everything.

**Comment by justinGilmer** [comment]
And at the moment we are targeting Journal of Chemical Engineering Data (JCED). So we can also edit the rc for that.

**Comment by justinGilmer** [comment]
The `matplotlibrc` file from @daico007 's gist has been included in #72
